### ISSUE TITLE
Can't give you my money

### ISSUE THREAD

**Issue by dancyfits** [monicahq/monica#368]
Trying to upgrade my account leads to a page that says "Whoops, looks like something went wrong."  Take my money, dammit!

Side note: I'm more than happy to give 10 bucks a month to help, as this project is something I'm REALLY looking forward to taking advantage of.  That said, 10 bucks a month seems a little steep to me, at least at this stage of development.  Just my 2 cents 😄.  Any reasoning behind that price in particular?

**Comment by dancyfits** [comment]
Actually, just got a call from my bank asking if it was fraudulent activity.  I'm guessing it's a newly opened stripe account?

**Comment by djaiss** [comment]
Damn!!!! I didn't even have had an error on the server... Yes, it's a newly created account on Stripe :-(

**Comment by djaiss** [comment]
The pricing is a test actually. You are probably right about the amount.

**Comment by djaiss** [comment]
I'm going to fix this first thing in the morning. So sorry about this.

**Comment by dancyfits** [comment]
Haha, sorry you can't take my money?  No worries.  Keep up the good work.

**Comment by Phrohdoh** [comment]
@dancyfits What would you consider to be a more reasonable price? Or on the other side... what would you like to see in the product to justify that price?

(I don't want to imply that $10 is _the_ price because as Régis said it is a test value and I don't have a seat to determine price, I'm just trying to get directed feedback)

**Comment by djaiss** [comment]
@dancyfits yeah so Stripe says that the bank declined the transfer and that you have to ask for the payment to be processed.

First of all I need to catch the exception in the code.

Secondly, I need to check with Stripe if newly created accounts usually have this kind of errors.

**Comment by dancyfits** [comment]
@Phrohdoh I have no idea.  My brain is popping out "$3.99!".  Totally arbitrary number.  Maybe a community poll would be helpful?  The more I think about it, at this stage in development it's less about getting the extra features and more about supporting the project.  Perhaps a Patreon page in the meantime?

@djaiss Yup, that's annoying.  I did tell my bank that the charge was legit; I'm betting if I try again it'll go through.

When I talked to the lady at my bank she sounded confused when saying the charge was from "Monica", as if I found a 10 dollar hooker or something 😆

**Comment by djaiss** [comment]
@dancyfits [Patreon page](https://www.patreon.com/monicahq) is up already, for those who want to support the project and have self hosted instances.

Yeah, the name has some issues here and there, I agree, but I also like it because it's memorable.

**Comment by dancyfits** [comment]
Finally got around to attempting to subscribe again; all is good!

As for the name, I still can't believe [Soylent](https://www.soylent.com) even thought that was a good brand name in the first place, yet somehow I now think of the meal replacement drink before I think "soylent green is people".  It seems there's a point where the branding can outweigh preconceived notions if the product is good enough, and I think Monica is a strong enough concept.  That said, I have a hard time imagining what the digital story for something called "Monica" would be.

I was just wondering why the subscription is called the Chandler plan and then realized that Monica was a character on Friends as well.  What... what is the motivation here...?

**Comment by djaiss** [comment]
@dancyfits 

> What... what is the motivation here...?

At first Monica was my own internal codename, as a reference to one of my favorite tv show (and there is another reason as well but wtv), then when I wanted to buy a domain name, everything was taken, except everything around Monica. So I just took it. Then the project caught on. That's it.


Thanks a lot for your subscription. By the way I noticed that other people have tried to subscribe as well without success. I need to fix that, because there is definitely another problem here. Damn damn.
